### PR TITLE
Fix typo on welcome page

### PR DESCRIPTION
**Pull request by jairoprez**
Fixes https://github.com/Strategy11/formidable-pro/issues/2913

### CODE REVIEW

File: classes/views/welcome/show.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
 					?>
 				</div>
 				<h1><?php esc_html_e( 'Welcome to Formidable Forms!', 'formidable' ); ?></h1>
-				<p><?php esc_html_e( 'Thanks for choosing Formidable Froms - The most powerful and vesatile form builder for Wordpress', 'formidable' ); ?></p>
+				<p><?php esc_html_e( 'Thanks for choosing Formidable Froms - The most powerful and versatile form builder for Wordpress', 'formidable' ); ?></p>
```
Comment (Crabcyborg) [inline classes/views/welcome/show.php:21]: `Formidable Froms` too 😂

Comment (Crabcyborg) [reply classes/views/welcome/show.php:21]: Oh, also, it's `WordPress`. I think it's good to get the capitalization right.

Comment (jairoprez) [reply classes/views/welcome/show.php]: @Crabcyborg Looks like there were still other typos lurking :sweat_smile: 
Nice catch - Thanks!

**Review by Crabcyborg — APPROVED** [review]
Thank @jairoprez! Looks good!

**Review by stephywells — APPROVED** [review]
Doh!

**Pull request merged**